### PR TITLE
Remove unique number and update test

### PR DESCRIPTION
**Pull request by liamtoozer**
# Motivation and Context

Unique number is no longer part of the uac-link table so needs to be removed.

# What has changed

* Removed column from uac-qid link table

# How to test?

Run acceptance tests

# Links

https://trello.com/c/cXw970Gx/902-build-uac-not-iac-qid-service-8#

### CODE REVIEW

**Comment by liamtoozer** [comment]
[Build UAC (not IAC!)/QID Service (8)](https://trello.com/c/cXw970Gx/902-build-uac-not-iac-qid-service-8)

File: .travis.yml [hunk 1]
```diff
@@ -25,3 +30,4 @@ cache:
 branches:
   only:
     - master
+    - model-update
```
Comment (AdamHawtin) [inline .travis.yml]: Shouldn't be set to branch build, it'll be building twice for every commit while the PR is open.

File: .travis.yml [hunk 2]
```diff
@@ -17,6 +17,11 @@ after_success:
     docker login -u "${DOCKER_GCP_USERNAME}" -p "${DOCKER_GCP_PASSWORD}" "${DOCKER_GCP_REGISTRY}";
     docker push "eu.gcr.io/census-rm-ci/rm/census-rm-case-api";
     fi
+  - if [ "$TRAVIS_BRANCH" == "model-update" ] ; then
```
Comment (AdamHawtin) [inline .travis.yml]: Needs removing.

File: src/test/resources/docker-compose.yml [hunk 3]
```diff
@@ -25,7 +25,7 @@ services:
 
   case-processor:
     container_name: case-processor-it
-    image: eu.gcr.io/census-rm-ci/rm/census-rm-case-processor
+    image: eu.gcr.io/census-rm-ci/rm/census-rm-case-processor:ade_test
```
Comment (AdamHawtin) [inline src/test/resources/docker-compose.yml]: Testing tag committed?

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**